coral/src/app/router.tsx

```diff
@@ -349,7 +356,118 @@ const routes: Array<RouteObject> = [
   },
 ];
 
-const router = createBrowserRouter(routes, {
+/** `superadminRouteMap` is used as in-between-solution
```

Comment:
I decided to leave the map itself here in `router.tsx` because it's similar to our `routes` map and I thought it's good to keep them together? 🤔